test/promtool_test.lua

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env tarantool
+local fio = require('fio')
+
+local metrics = require('metrics')
+local counter = metrics.counter('counter_total', 'help text')
+local gauge = metrics.gauge('gauge', 'help text')
+local histogram = metrics.histogram('histogran', 'help text', nil)
+local summary = metrics.summary('summary', 'help text', {[0.5] = 0.01, [0.9] = 0.01, [0.99] = 0.01})
+for i = 1, 10^3 do
+    summary:observe(i)
+    histogram:observe(i)
+end
+counter:inc(1)
+gauge:set(1)
+local httpd = require('http.server')
+local http_handler = require('metrics.plugins.prometheus').collect_http
```

Comment:
is there a more straightforward way to get prometheus formatted metrics? I mean without setting up http server